doc/api/http.md

```diff
@@ -102,9 +119,9 @@ added: v0.3.4
   Can have the following fields:
   * `keepAlive` {Boolean} Keep sockets around in a pool to be used by
```

Comment:
This doesn't really describe the pooling condition of "no outstanding requests", I suggest:

> Keep sockets in a pool even when there are no outstanding requests, so it can be used for future requests without having to reestablish an HTTP connection.